storm-core/src/jvm/backtype/storm/package-info.java

```diff
@@ -0,0 +1,7 @@
+/*
```

Comment:
This file is also a build artifact and shouldn't be checked in.  Please add it to the .gitignore file.  I would prefer to have it placed in a different directory from the rest of the source code.  The maven plugin org.codehaus.mojo:build-helper-maven-plugin is a good way to add in a generated source directory using the add-source goal.  It also allows us to not include a checksum of this generated file in the checksum of the source.